server/sonar-web/src/main/js/apps/organizations/components/OrganizationMembers.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+// @flow
+import React from 'react';
+import PageHeader from '../../users/components/PageHeader';
+import UsersSearch from '../../users/components/UsersSearch';
+import UsersList from '../../users/components/UsersList';
+import UsersListFooter from '../../users/components/UsersListFooter';
+import type { Organization } from '../../../store/organizations/duck';
+import type { Member } from '../store/membersDuck';
+
+type Props = {
+  members: Array<Member>,
+  state: { loading?: boolean, total?: number, pageIndex?: number, query?: string },
+  organization: Organization,
+  fetchOrganizationMembers: (organizationKey: string, query?: string) => void,
+  fetchMoreOrganizationMembers: (organizationKey: string, query?: string) => void
+};
+
+type State = {
+  query: string | null
+};
+
+export default class OrganizationMembers extends React.PureComponent {
+  props: Props;
+
+  state: State = {
+    query: null
```

Comment:
When I want to load more members, I need to know what was the last search to load more matching members.